changelogs/unreleased/changelogs/unreleased/8304-update-installation-instructions-native-docker-container.yml

```diff
@@ -0,0 +1,10 @@
+description: Update installation instructions for new native docker container
+issue-repo: inmanta-core
+issue-nr: 8304
+change-type: patch
+destination-branches: [master]
+sections:
+  upgrade-note: >
+    The image inmanta/orchestrator:8 introduces a number of changes, including these breaking changes:
```

Comment:
This version number isn't correct. OSS works with `<year>.<release-nr>` versioning. The next release will be `2025.1`.

Reply:
Interesting, I was completely unaware of that.